frontend/src/components/grantaccess/UserIdentityCard.vue

```diff
@@ -1,24 +1,32 @@
 <script setup lang="ts">
+import { toRaw } from 'vue';
 import type { IdimProxyIdirInfo } from 'fam-api';
 import Card from 'primevue/card';
+import Icon from '../common/Icon.vue';
+import { IconSize } from '@/enum/IconEnum';
 
 const props = defineProps<{
     userIdentity: IdimProxyIdirInfo;
 }>(); // Vue3 alternative way for Type the defineProps().
+console.log(toRaw(props.userIdentity))
```

Comment:
let'e remove the console log

Reply:
sorry, I thought I had removed the console.logs.